test/extended/openstack/servers.go

```diff
@@ -0,0 +1,186 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/blang/semver"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/flavors"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/images"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/stretchr/objx"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+)
+
+const (
+	machineAPINamespace   = "openshift-machine-api"
+	machineLabelRole      = "machine.openshift.io/cluster-api-machine-role"
+	machineAPIGroup       = "machine.openshift.io"
+	machineSetOwningLabel = "machine.openshift.io/cluster-api-machineset"
+)
+
+var _ = g.Describe("[sig-installer][Feature:openstack] The OpenStack platform", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLI("openstack")
+
+	g.BeforeEach(func() {
+		SkipUnlessOpenStack(context.TODO(), oc)
+	})
+
+	g.Context("while instance creation", func() {
```

Comment:
nit:
I think that should sounds better as "on instance creation" rather than "while instance creation"

But it's really a nit and feel free to ignore :)